cli/testflinger_cli/__init__.py

```diff
@@ -541,6 +541,22 @@ def submit_job_data(self, data: dict, headers: dict = None):
                     "Received 404 error from server. Are you "
                     "sure this is a testflinger server?"
                 )
+            if exc.status == 401:
+                sys.exit(
+                    "Received 401 error from server. You are "
+                    "attempting to use a feature that requires "
+                    "client authorisation without using client "
+                    "credentials. See https://testflinger.readthedocs"
+                    ".io/en/latest/how-to/authentication/ for more details"
+                )
+            if exc.status == 403:
+                sys.exit(
+                    "Received 403 error from server with reason "
+                    f"{exc.msg}The specified client credentials do "
```

Comment:
```suggestion
                    f"{exc.msg}"
                    "The specified client credentials do "
```

Reply:
Yeah I did that since exc.msg already had a newline character at the end for some reason. I can do that change for readability.